OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -212,6 +212,7 @@ public interface IOccupySpace
 		WPos CenterPosition { get; }
 		CPos TopLeft { get; }
 		IEnumerable<Pair<CPos, SubCell>> OccupiedCells();
+		IEnumerable<Pair<CPos, SubCell>> TargetableCells();
```

Comment:
Please split this to its own interface.  `IOccupySpace` is part of our lowest-level ActorMap code, which should remain separate from targeting.